### ISSUE TITLE
ARIA should be a last resort

### ISSUE THREAD

**Issue by oliverjam** [fac24/week2-Minju-Hussain-Asmahan-George#35]
I noticed you've added a lot of `aria-label`s to inputs that already have visible labels. This is generally not a good idea, since users of assistive tech should be able to access the visual labels fine. More info in this issue on another team's repo:

https://github.com/fac24/week2-Bereket-Hallie-Joe-Petra/issues/37
